### PR TITLE
add logical-import-prep-stmt

### PR DESCRIPTION
**Pull request by dbsid**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/18902
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qw4990 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -227,6 +227,9 @@ strategy = ""
 # 你可以减小该值以减轻大事务对集群的压力。
 # logical-import-batch-rows = 65536
 
+# 在逻辑导入模式下，该参数控制是否使用预处理语句和语句缓存来提高性能。
+# logical-import-prep-stmt = false
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:231]:
```suggestion
logical-import-prep-stmt = false
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -227,6 +227,9 @@ strategy = ""
 # 你可以减小该值以减轻大事务对集群的压力。
 # logical-import-batch-rows = 65536
 
+# 在逻辑导入模式下，该参数控制是否使用预处理语句和语句缓存来提高性能。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-configuration.md:230]:
```suggestion
# 在逻辑导入模式下，该参数控制是否使用预处理语句和语句缓存来提高性能。默认值为 false。
```

**Review by hfxsd — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -227,6 +227,9 @@ strategy = ""
 # 你可以减小该值以减轻大事务对集群的压力。
 # logical-import-batch-rows = 65536
 
+# 在逻辑导入模式下，该参数控制是否使用预处理语句和语句缓存来提高性能。默认值为 false。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:230]:
```suggestion
# 在逻辑导入模式下，该参数控制是否使用预处理语句和语句缓存来提高性能。默认值为 `false`。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**